src/app/work-item/work-item-list/work-item-detail/work-item-detail.component.ts

```diff
@@ -98,11 +99,15 @@ export class WorkItemDetailComponent implements OnInit {
             .getUser()
             .then(user => this.setLoggedInUser(user));
         }
+        //set the creator/reporterUser
```

Comment:
Move these three lines before line no 96. In **this.setLoggedInUser** the logged in user is removed from the all user list. This will cause a potential bug, where if the creator and logged in user is same, the code won't find the creator in the user list.